core/src/test/scala/blobstore/AbstractStoreTest.scala

```diff
@@ -279,61 +299,59 @@ trait AbstractStoreTest extends AnyFlatSpec with Matchers with BeforeAndAfterAll
       _ <- store.remove(dstDir / "filename.txt")
       _ <- store.remove(srcDir / "filename.txt")
     } yield {
-      c1.mkString("\n") must be(c2.mkString("\n"))
+      c1 mustBe c2
     }
 
     test.unsafeRunSync()
   }
 
-  // TODO this doesn't test recursive directories. Once listRecursively() is implemented we can fix this
-  it should "remove all should remove all files in a directory" in {
-    val srcDir = dirPath("rm-dir-to-dir-src")
+  it should "remove all files in a directory" in {
+    val srcDir = dirUrl("rm-dir-to-dir-src")
 
     (1 to 10).toList
       .map(i => s"filename-$i.txt")
-      .map(writeFile(store, srcDir))
+      .map(writeFile(store, srcDir.path))
```

Comment:
This also missing deeper levels of recursion. I think adding sub-folders breaks FileStore and BoxStore.